### PR TITLE
Add short alias for authorization policy resources

### PR DESCRIPTION
**Pull request by rolandkool**
Suggested fix for https://github.com/istio/istio/issues/19292

### CODE REVIEW

File: install/kubernetes/helm/istio-init/files/crd-all.gen.yaml [hunk 1]
```diff
@@ -96,6 +96,8 @@ spec:
     kind: AuthorizationPolicy
     plural: authorizationpolicies
     singular: authorizationpolicy
+    shortNames:
+    - authzpol
```
Comment (yangminzhu) [inline install/kubernetes/helm/istio-init/files/crd-all.gen.yaml]: Thanks for the PR, I'm wondering can we just make it `authz`?

Comment (rolandkool) [reply install/kubernetes/helm/istio-init/files/crd-all.gen.yaml]: Fine with me. I choose authzpol as this resource seems a bit similar to network policies. It is a list, so we could add both?

Comment (yangminzhu) [reply install/kubernetes/helm/istio-init/files/crd-all.gen.yaml]: Yeah, I think we can do both.

**Comment by rolandkool** [comment]
@googlebot I signed it!

**Comment by howardjohn** [comment]
@rolandkool not sure what is going on. I reached out to the CLA team to see whats happening

**Comment by howardjohn** [comment]
@rolandkool they said the CLA was denied. Contact me on slack or email if you need more info

**Comment by rolandkool** [comment]
@googlebot I signed it!

**Comment by rolandkool** [comment]
@googlebot I signed it!

**Comment by howardjohn** [comment]
This file is autogenerated from the API repo, so it shouldn't be directly modified like this. Can you submit this there if this is still desired?

**Pull request closed without merging**